server/src/domain/job/job.service.ts

```diff
@@ -28,6 +29,7 @@ export class JobService {
     @Inject(IPersonRepository) private personRepository: IPersonRepository,
   ) {
     this.configCore = SystemConfigCore.create(configRepository);
+    this.configCore.addValidator((config) => this.validateCronExpression(config.library.scan.cronExpression));
```

Comment:
The cron expression is for the library service, so it should be the one subscribing to updates and throwing an error for an invalid configuration.